x-pack/legacy/plugins/siem/public/components/inspect/translations.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const TOOLTIP_CONTENT = i18n.translate('xpack.siem.inspect.tooltipDescription', {
+  defaultMessage: 'Inspect',
+});
+
+export const TITLE = i18n.translate('xpack.siem.inspect.modalTitle', {
+  defaultMessage: 'Inspect',
+});
```

Comment:
You have two of the same names, but separated by different keys?

Letting you know in case that is not intentional. Otherwise, ignore this comment.